third_party/2and3/nmap/nmap.pyi

```diff
@@ -0,0 +1,127 @@
+import sys
+from typing import Callable, Dict, Iterator, List, Optional, Sequence, Text, Tuple, TypeVar
+
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+else:
+    from typing_extensions import TypedDict
+
+_T = TypeVar("_T")
+_Callback = Callable[[str, _Result], None]
```

Comment:
Since the return value is ignored by the implementation, and therefore doesn't matter, it should be `Any`, not `None`.

Reply:
Okay, taken.